plugin/README.md

```
@@ -0,0 +1,45 @@
+```
+/plugin marketplace add kennethreitz/pytheory-skill
+/plugin install composing-with-pytheory@pytheory
+```
+
+Then just ask for music — *"make a four-chord pop loop"*, *"lay down a funk
+beat with ghost-note snares"*, *"turn ii–V–I in D minor into a short
+arrangement and save the MIDI"*.
+
+You'll also want PyTheory itself available where Claude runs:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)